src/main/java/ua/hillel/controller/OrderController.java

```diff
@@ -0,0 +1,39 @@
+package ua.hillel.controller;
+
+import jakarta.servlet.http.HttpServlet;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ua.hillel.OrderRepository;
+import ua.hillel.entity.Order;
+
+import java.util.List;
+
+
+@RequestMapping("api/orders")
+@RestController
+public class OrderController extends HttpServlet {
```

Comment:
You do not need to extend from http servlet if you use Spring.

Reply:
+++